maps/outreach/outreach_exoplanet.dm

```diff
@@ -1,69 +1,224 @@
+/datum/map/outreach/build_exoplanets()
+	log_debug("MAP: building exoplanets!") //#REMOVEME
+	if(!outreach_initialized)
+		for(var/z in global.overmap_sectors)
+			var/obj/effect/overmap/visitable/sector/exoplanet/outreach/O = global.overmap_sectors[z]
+			if(istype(O))
+				O.build_level() //We have to force outreach to update now, otherwise it won't set its atmosphere
+				outreach_initialized = TRUE
+				break
+	. = ..()
+
+/**Returns a list with the ratio (0 to 1) for each gases in the atmosphere of outreach. */
+/proc/OutreachAtmosRatios()
+	return list(
+		/decl/material/gas/oxygen         = 0.10,
+		/decl/material/gas/chlorine       = 0.17,
+		/decl/material/gas/nitrogen       = 0.53,
+		/decl/material/gas/carbon_dioxide = 0.11,
+		)
+
 /obj/effect/overmap/visitable/sector/exoplanet/outreach
-	name = "\improper Outreach"
-	desc = "A mining border-world, home to those lost in space."
-	planetary_area = /area/exoplanet/outreach
-	daycycle = 25 MINUTES
-	daycycle_column_delay = 10 SECONDS
-	night = TRUE
-	daycolumn = 1
+	name                  = "\improper Outreach"
+	desc                  = "A barren mining world covered in chlorine deserts, home to those lost in space."
+	color                 = "#c9df9f"
+	planetary_area        = /area/exoplanet/outreach
+	daycycle              = 30 MINUTES
+	night                 = FALSE
+	lightlevel            = 0.7
+	starlight_color       = COLOR_YELLOW_GRAY
+	rock_colors           = list(COLOR_GRAY80, COLOR_PALE_GREEN_GRAY, COLOR_PALE_BTL_GREEN)
+	plant_colors          = list(COLOR_PALE_PINK, COLOR_PALE_GREEN_GRAY, COLOR_CIVIE_GREEN)
+	surface_color         = COLOR_PALE_GREEN_GRAY
+	water_color           = COLOR_BOTTLE_GREEN
+	water_material        = /decl/material/gas/chlorine
+	ice_material          = /decl/material/gas/chlorine
 
-	rock_colors = list(COLOR_GRAY80, COLOR_PALE_GREEN_GRAY, COLOR_PALE_BTL_GREEN)
-	plant_colors = list(COLOR_PALE_PINK, COLOR_PALE_GREEN_GRAY, COLOR_CIVIE_GREEN)
-	surface_color = COLOR_PALE_GREEN_GRAY
-	water_color = COLOR_BOTTLE_GREEN
-	crust_strata = /decl/strata/sedimentary
+	//Sector Access
+	free_landing          = FALSE
```

Comment:
This needs to be TRUE, otherwise nothing can land and take off from the surface.

Reply:
Speaking of that. I really don't think we should let shuttles land anywhere on the planet, but on docking areas?

Because the shuttle code still lets you land on any z-levels of the planet, and pretty much anywhere without an area protection thing. 
That in turn is going to mess with the base turfs when the thing takes off. And I just can't guarantee it won't cause problems or end up being used for griefing.

For reference, landing on a transition edge is going to permanently fuck the map. The underground turfs ended up being less trouble being simulated turfs for allowing laying down cable and etc,. So a shuttle landing there is very bad..

I can turn it back on, but like, can we at least add support for docking ports, so we don't have to deal with shuttle damage again? It's really hard to recover anything that a shuttle destroys.